docs/content/v0.20/examples.md

```diff
@@ -139,14 +139,6 @@ spec:
 EOF
 ```
 
-Verify that *manifestworks* wrapping the objects have been created in the mailbox
-namespaces:
-
-```shell
-kubectl --context imbs1 get manifestworks -n cluster1
-kubectl --context imbs1 get manifestworks -n cluster2
-```
```

Comment:
I don't think we need to remove this. This has no number enforcement

Reply:
The question is whether checking manifestworks should be externalized?  Is this something a user should be aware of?

Reply:
Following our discussion, comment accepted and text restored.